report/src/sections/01-introduction.tex

```diff
@@ -1,9 +1,10 @@
 \section{Introduction}\label{sec:introduction}
-\IEEEPARstart{S}{oftware} analysis is a broad field encompassing a wide range of techniques and methodologies, each tailored to address specific aspects of software systems.
-One such technique is value analysis, which focuses on the behavior of a program with respect to its input and output values~\cite{jackson_software_2000}.
-Value analysis can be approached from various angles, including the consideration of meta-information like information flow or taint analysis.
-While these approaches offer valuable insights into the behavior of a program, they often require detailed knowledge of the program's internal workings and are not always applicable to all types of software systems.
-In this paper we use static program analysis, specifically abstract interpretation.
+\IEEEPARstart{S}{oftware} analysis is a broad field encompassing various techniques and methodologies, each tailored to address specific aspects of software systems.
+One such technique is sound analysis, which focuses on finding information guaranteed to hold during execution~\cite{jackson_software_2000}.
+Two common approaches to sound analysis are dynamic and static analysis.
+Dynamic analysis finds information guaranteed to hold at a specific execution, while static analysis involves finding information guaranteed to hold at all executions.
+While the dynamic approach offers valuable insights into the specific behaviour of a program's single execution, static analysis provides a more comprehensive view by ensuring that certain properties hold in all possible executions.
```

Comment:
What we are doing is called static analysis and more specifically data flow analysis.